src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3181,6 +3181,12 @@ protected BoundExpression BindObjectCreationExpression(ObjectCreationExpressionS
         {
             var type = BindType(node.Type, diagnostics);
 
+            if (node.Type.Kind() == SyntaxKind.TupleType)
```

Comment:
Can you please move this constraint into the parser? It is likely to be specified as a syntactic constraint. The syntax `new (x, y)` is reasonably likely to be a work item in the not too distant future (#35). Having this handled in the parser would make the next person's job much easier.